### ISSUE TITLE
Add function to return the directory `just` was invoked from

### ISSUE THREAD

**Issue by joe776** [casey/just#312]
I would love to see the option to run the commands in the current directory instead of always from the directory where the justfile is located.
For example, I'm using just in a nested Maven project and sometimes I just want to run the commands on a submodule instead of the whole project, but the justfile is at the top level.
This could be an option, maybe by prefixing the command with a dot (either in the justfile itself or from the shell).

**Comment by casey** [comment]
Thank you for the feature request!

Can you give some examples of commands you'd like to run? I don't use Maven, so I want to be sure I understand the specific use case.

**Comment by xitep** [comment]
hello, i'd welcome such a possibility as well. my motivation would be: i usually have `Justfiles` in the root of my projects, defining amongst others some project specific, frequently used commands/aliases like:

```
import-web file country='7' language='en':
        @curl -D /dev/stderr -X POST "http://localhost:8080/{{country}}/{{language}}/foobar" -H  "accept: application/json" -H  "content-type: multipart/form-data" -F "file=@{{file}};type=text/xml"

import-docker file:
        @docker cp "{{file}}" docker_ekomi-import_1:/data
```

this allows me to easily maintain per project specific command-line "shortcuts" which proved very useful.

now, with the above example, when invoking `just import-web ../imports/abc.csv` or `just import-docker abc.csv` (ie. specifying the "file" argument as a relative pathname) from within a sub-directory, the receipts typically fail (because the relative path is not right any more after `just` switches the working directory) and i have to retry with an absolute pathname or from within the directory hosting the `Justfile`. i hope this illustrates one use-case.

**Comment by joe776** [comment]
My use case is similar, but the commands are simpler. We have a `parent` project (with _justfile_) and therein some Maven modules, for example multiple `backend`s and `web` depending on those. When developing and testing locally and something in one of the backends changes, it needs to be rebuilt and the web module be restarted. The recipe run from within any of the modules would ideally look like this:
```
inst:
        ../web/stop_jetty.sh
        mvn clean install -DskipTests=true
        ../web/start_jetty.sh
```
Currently, the `install` would however run in the parent module andbuild everything (and the path to the `web` module is wrong).

Another solution would be to provide the directory from which _just_ was called as a built-in function like `arch()`, e.g. `pwd()`. Then I could change the recipe to 
```
inst:
        web/stop_jetty.sh
        mvn -f {{pwd()}}/pom.xml clean install -DskipTests=true
        web/start_jetty.sh
```
and Maven would always build according to the configuration in the current submodule.

**Comment by runeimp** [comment]
I really like the `pwd()` function idea. Though I'd like there to also be a `pwd_safe()` function as well. The difference being that `pwd_safe()` would ask the user something like `Your about to change the working directory from /foo to /foo/bar. Is this OK? [Y/n] `. That way if I forgot the recipe changed the working directory or that the directory I call the recipe from is important then I have the option of opting out of a potentially very destructive process. Especially if that is run by another dev who is not familiar with the recipes yet and just learning the how to work with the project.

**Comment by casey** [comment]
Fixed by #323, thanks to @joshuawarner32! I'll cut a release now with the new functionality.
